cg/cli/deliver/base.py

```diff
@@ -85,3 +87,57 @@ def rsync(context: CGConfig, ticket_id: int, dry_run: bool):
     """
     rsync_api = RsyncAPI(config=context)
     rsync_api.run_rsync_command(ticket_id=ticket_id, dry_run=dry_run)
+
+
+@deliver.command(name="concatenate")
```

Comment:
is concatenate ever used by itself? Perhaps it doesnt need its own cli command?

Reply:
Production will need it separately for certain situations